### PR TITLE
Update About page

### PR DESCRIPTION
**Pull request by schamp**

### CODE REVIEW

**Comment by schamp** [comment]
Feel free to review but don't merge pending stamp by Mikayla on https://trello.com/c/dUg7wEY4/238-can-about-page-updates

**Comment by schamp** [comment]
Mikayla confirms the design is right, so just looking for code review now.

File: src/cms-content/about/index.tsx [hunk 1]
```diff
@@ -30,8 +30,6 @@ interface AboutContent {
   partnersContent: PartnersContent[];
   teamHeader: string;
   teamIntro: Markdown;
-  futureProjectsHeader: string;
-  futureProjectsContent: Markdown;
```
Comment (mikelehen) [inline src/cms-content/about/index.tsx:34]: We should remove these from the CMS config too if we no longer support them: https://github.com/covid-projections/covid-projections/blob/develop/public/admin/config.yml#L98

File: src/screens/About/About.tsx [hunk 2]
```diff
@@ -141,15 +137,9 @@ const About = () => {
             <MarkdownContent source={aboutContent.impactContent} />
           </SectionContent>
           <ButtonContainer>
-            <Button to="/case-studies" cta="View case studies" />
+            <Button href="https://actnowcoalition.org" cta="Learn more" />
```
Comment (mikelehen) [inline src/screens/About/About.tsx:140]: Nit: Can you link to https://www.actnowcoalition.org/ instead (with www) to avoid a redirect?

**Review by mikelehen — APPROVED** [review]
Looks good with a couple nits.

**Comment by schamp** [comment]
Can do, thanks

**Pull request merged**